scripts/data/supersearch_fields.json

```diff
@@ -3450,7 +3386,7 @@
         "query_type": "enum",
         "permissions_needed": [],
         "storage_mapping": {
-            "type": "string"
+            "type": "integer"
```

Comment:
According to https://crash-stats.mozilla.com/search/?_facets=support_classification that field contains only integers.